### DISCUSSION TITLE
move the badges with paragraphs and songs to the thumbnail

### DISCUSSION THREAD

**Discussion by dagandrew** [sircharlo/meeting-media-manager#2765] [Ideas] [answered]
(with a background, so it could be visible) I have this issue because I use a very small window and the numbers are taking so much space there
![image](https://github.com/user-attachments/assets/2e9c3407-2654-4fbb-a034-7ec58bc802bb)

**Comment by mtdvlpr** [comment, marked as answer]
Duplicate of #2937
